index.html

```diff
@@ -2,394 +2,275 @@
-              <img
-                class="counter"
-                src="imgs/counter.png"
-                alt="Back End Dev"
-              />Back End Dev
+            <li>
+              <img src="img/html-icon.png" alt="javascript" />
+              <p>HTML</p>
             </li>
-            <li class="year">
-              <img class="counter" src="imgs/counter.png" alt="2015" />2015
+            <li>
+              <img src="img/css-icon.png" alt="javascript" />
+              <p>CSS</p>
             </li>
           </ul>
-        </section>
-
-        <section class="description-4">
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-        </section>
-        <section class="programs">
-          <ul>
-            <li class="sty-1">html</li>
-            <li class="sty-2">css</li>
-            <li class="sty-3">javaScript</li>
-          </ul>
-        </section>
-
-        <section class= "see-button-4">
-          <button class="see-btn">See Project</button>
-        </section>   
-
-      </section>
-      
-      <section class="pg6">
-        <section class="self">
-          <h2>
-            About <br />
-            Myself
-          </h2>
-        </section>
-
-        <section class="des">
-          <p>
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
-        </section>
-
-        <section class="let">
-          <h4>LET'S CONNECT</h4>
-        </section>
-
-        <section class="apps">
-          <ul>
-            <li class="ap-1">
-              <a
-                href="https://twitter.com/abuyiag"
-                target="_blank"
-                rel="noopener noreferrer"
-                ><img src="imgs/cat.png" alt="cat chat"
-              /></a>
+        </ul>
+        <li class="frameworks">
+          <div class="user-tool-d">
+            <h3>Frameworks</h3>
+            <i class="fas fa-chevron-right"></i>
+          </div>
+          <hr />
+        </li>
+        <li class="skills">
+          <div class="user-tool-d">
+            <h3>Skills</h3>
+            <i class="fas fa-chevron-right"></i>
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form">
+      <div class="contact-form-holder">
+        <h2 class="contact-title">Contact Me</h2>
+
+        <p class="p-pargraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a work-section-li that needs coding. I’d love to help
+          with it
+        </p>
+
+        <form
+          action=" https://formspree.io/f/xqkwojbj"
+          class="form-contact"
+          method="post"
+        >
```

Comment:
- [ ] _[OPTIONAL]_ The `Contact me` section is missing some background images, adding them will make your portfolio look much more appealing.

- [ ] On both screens, your `contact me` form is a bit different than the Figma design(some input fields are not styled, and the `Get in touch` button should be centered), kindly make sure that your form matches the Fifma design as much as you can.
Your design
![image](https://user-images.githubusercontent.com/33767951/140610481-fa534470-8fc8-42a8-a785-932dceeb9252.png)
Figma design
![image](https://user-images.githubusercontent.com/33767951/140610634-4e148529-b246-4839-8074-6ec796929383.png)

Reply:
Proper styling has been added to the form

Reply:
-Proper styling of the form section.

Reply:
-black border around input fields has been removed.
-Breaking form has been rectified.